Engine/Rumor.cs

```diff
@@ -74,6 +74,7 @@ public Node Next
 		/// null if no choices have been chosen yet.
 		/// </summary>
 		public string LastChoice { get; set; }
+		public int LastChoiceIndex { get; set; }
```

Comment:
I don't think adding `LastChoiceIndex` is a good idea. The index doesn't seem very useful after it has already been removed, so why do you want to be able to know what it was?

Reply:
Yeah I was procedurally generating the choice strings in script and would have had to save off the strings in the game to determine which choice (0, 1, or 2) was taken. This seemed like the easiest way to do it, but is not necessarily the cleanest. I'll just remove it from the PR.